great_expectations/core/expectation_configuration.py

```diff
@@ -1419,6 +1424,30 @@ def get_domain_type(self) -> MetricDomainTypes:
             'Unable to determine "domain_type" of this "ExpectationConfiguration" object from "kwargs" and heuristics.'
         )
 
+    def render(self) -> None:
+        """
+        Renders content using the atomic prescriptive renderer for this Expectation Configuration to
+            ExpectationConfiguration.rendered_content.
+        """
+        inline_renderer_config: "InlineRendererConfig" = {  # type: ignore[assignment]
+            "class_name": "InlineRenderer",
+            "render_object": self,
+        }
+        module_name = "great_expectations.render.renderer.inline_renderer"
+        inline_renderer = instantiate_class_from_config(
```

Comment:
Could we just instantiate from the constructor e.g. `inline_renderer = InlineRenderer(render_object=self)`? Or is this a point of extensibility where we would accept a config?

Reply:
This is the issue with circular imports that we spent a lot of time trying to solve about a month ago and landed on this as a solution for now.

Reply:
Would you mind adding a small comment here in the code with that explanation? That way if we fix the circular import issue in the future we will have a hint that we can safely remove the `instantiate_class_from_config()` call